draft-ietf-ppm-dap.md

```diff
@@ -1715,6 +1724,39 @@ following checks:
 
 If all of the above checks succeed, the input share is not marked as invalid.
 
+#### Report ID commitments {#report-id-commitments}
+
+Replay protection is implemented using "report ID commitments". A report ID
+commitment is an interval of report IDs (using lexicographic ordering),
```

Comment:
An interval of time makes sense, but an interval of report IDs is a strange notion.
```suggestion
commitment is an ordered set of report IDs (using lexicographic ordering),
```